### PR TITLE
add note for podman rootful mode

### PR DESCRIPTION
**Pull request by manmeetkaur**
https://astronomer.slack.com/archives/CL44PA4DB/p1682598067304229?thread_ts=1682358058.308999&cid=CL44PA4DB

### CODE REVIEW

File: astro/cli/configure-cli.md [hunk 1]
```diff
@@ -99,6 +99,11 @@ The Astronomer product you're using determines the format and behavior of the co
 
 
 ## Run the Astro CLI using Podman
+:::info 
+
+If you are using M1 Mac, you will need to turn on rootful mode for Podman using `podman machine set --rootful`.
```
Comment (lzdanski) [inline astro/cli/configure-cli.md:104]:
```suggestion
If you use an M1 Mac, turn on rootful mode for Podman by using `podman machine set --rootful`.
```
This is a style suggestion to make the composition active voice - if you're interested in learning more about why we do that in docs, would be happy to share!

**Review by lzdanski — CHANGES_REQUESTED** [review]
Small style detail

File: astro/cli/configure-cli.md [hunk 2]
```diff
@@ -99,6 +99,11 @@ The Astronomer product you're using determines the format and behavior of the co
 
 
 ## Run the Astro CLI using Podman
```
Comment (jwitz) [inline astro/cli/configure-cli.md:101]:
```suggestion
## Run the Astro CLI using Podman

```

**Review by jwitz — APPROVED** [review]
One comment about note placement, otherwise LGTM!

File: astro/cli/configure-cli.md (hunk 1 above)
Comment (manmeetkaur) [reply astro/cli/configure-cli.md]: by mistake I clicked on the wrong button, making these changes now.

**Review by lzdanski — APPROVED** [review]

**Pull request merged**